corehq/apps/reports/models.py

```diff
@@ -236,3 +225,18 @@ def create_jwt(self):
             }
         )
         return token
+
+
+class TableauGroup(models.Model):
+    server = models.ForeignKey(TableauServer, on_delete=models.CASCADE)
+    name = models.CharField(max_length=64)
+    group_id = models.CharField(max_length=32)
+
+
+class TableauUser(models.Model):
+    server = models.ForeignKey(TableauServer, on_delete=models.CASCADE)
+    username = models.CharField(max_length=64)
```

Comment:
How confident are you that 64 is long enough? Grepping, I see some username fields that are 128 or 255.

Reply:
I thought we didn't have a limit (our couch models are typically very loose about max lengths, to everyone's chagrin), but our users do tie to the django user model, so I think the max should be 150: https://docs.djangoproject.com/en/4.1/ref/contrib/auth/#django.contrib.auth.models.User.username